frontend/src/components/globalSearch/GlobalSearchContent.tsx

```diff
@@ -199,9 +199,11 @@ export function GlobalSearchContent({
     () =>
       filteredRoutes
         .filter(([, route]) => route.name && !route.path.includes(':'))
-        .filter(([, route]) => {
+        .filter(([key, route]) => {
           const clusterRoute = route.useClusterURL ?? true;
-
+          if (key === 'settingsCluster') {
```

Comment:
please add a little comment about this, cause it's not very clear why it's being filtered out 
something like `// settingsCluster is an old route that is just a redirect and shouldn't be included in the search results`